android/app/src/main/java/com/chorestory/app/ChildHomeActivity.java

```diff
@@ -0,0 +1,89 @@
+package com.chorestory.app;
+
+import android.os.Bundle;
+import android.support.design.widget.TabLayout;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+
+import com.chorestory.R;
+import com.chorestory.adapter.ChildHomeAdapter;
+import com.chorestory.fragment.ChildClanFragment;
+import com.chorestory.fragment.ChildFriendsFragment;
+import com.chorestory.fragment.ChildProfileFragment;
+import com.chorestory.fragment.ChildQuestsFragment;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ChildHomeActivity extends ChoreStoryActivity {
+
+    private final String CLAN = "Clan";
+    private final String FRIENDS = "Friends";
+    private final String QUESTS = "Quests";
+    private final String PROFILE = "Profile";
+
+    private final int[] tabSelectedIcons = {
+            R.drawable.castle_color,
+            R.drawable.friends_color,
+            R.drawable.sword_color,
+            R.drawable.knight_color
+    };
+
+    private final int[] tabsUnselectedIcons = {
```

Comment:
I think these 2 lists could go into a whole separate sub-module, like `icons` and get their whole function like:
`private final int[] menuBarIcons(int positionSelected)` that returns the specific list with `positionSelected` replaced by the colourful version.